regolith/fsclient.py

```diff
@@ -93,19 +95,39 @@ def load_json(self, db, dbpath):
             base, ext = os.path.splitext(collfilename)
             self._collfiletypes[base] = 'json'
             print('loading ' + f + '...', file=sys.stderr)
-            dbs[db['name']][base] = load_json(f)
+            coll = load_json(f)
+            dbs[db['name']][base] = coll
+            if base in self.chained_db:
+                for k, v in coll.items():
+                    if k in self.chained_db[base]:
+                        self.chained_db[base][k] = (self.chained_db[base][k].
+                            new_child(v))
```

Comment:
Is this correct? I think that the maps should be added in the same order that the databases are added.  As such, it is probably not best to add them in the loader function for a single database, but rather construct the chained_db after *all* databases are loaded.  Also, here, it seems that new_child() prepends the map: https://docs.python.org/3/library/collections.html#collections.ChainMap.new_child, we probably just want to append to the maps list.